storage_service/locations/models/package.py

```diff
@@ -289,6 +291,76 @@ def _update_quotas(self, space, location):
+
+        if self.current_location.purpose != destination_location.purpose:
+            raise ValueError('Location purpose mismatch for location %s' % destination_location_uuid)
+
+        self.status = Package.MOVING
+        self.save()
+
+        try:
+            # Move to internal storage
+            ss_internal = Location.active.get(purpose=Location.STORAGE_SERVICE_INTERNAL)
+
+            source_path = os.path.join(
+                self.current_location.relative_path,
+                self.current_path)
+
+            origin_space = self.current_location.space
+            origin_space.move_to_storage_service(
+                source_path=source_path,
+                destination_path='move',
+                destination_space=ss_internal.space)
+            origin_space.post_move_to_storage_service()
+
+            # Move to destination location
+            source_path = os.path.join(
+                'move',
+                os.path.basename(self.current_path))
+            destination_path = os.path.join(
+                destination_location.relative_path,
+                self.current_path)
+
+            destination_space = destination_location.space
+            destination_space.move_from_storage_service(
+                source_path=source_path,
+                destination_path=destination_path)
+            destination_space.post_move_from_storage_service(
+                staging_path=source_path,
+                destination_path=destination_path,
+                package=self)
+
+            # Update location
+            self.current_location = destination_location
+            self.status = Package.UPLOADED
+            self.save()
+            self.current_location.space.update_package_status(self)
+
+            # Update pointer file's location information
+            if self.pointer_file_path and self.package_type in (Package.AIP, Package.AIC):
+                root = etree.parse(self.full_pointer_file_path)
+                element = root.find('.//mets:file', namespaces=utils.NSMAP)
+                flocat = element.find('mets:FLocat', namespaces=utils.NSMAP)
+                if self.uuid in element.get('ID', '') and flocat is not None:
+                    flocat.set('{{{ns}}}href'.format(ns=utils.NSMAP['xlink']), self.full_path)
+                # Add USE="Archival Information Package" to fileGrp.  Required for
+                # LOCKSS, and not provided in Archivematica <=1.1
+                if root.find('.//mets:fileGrp[@USE="Archival Information Package"]', namespaces=utils.NSMAP) is not None:
+                    root.find('.//mets:fileGrp', namespaces=utils.NSMAP).set('USE', 'Archival Information Package')
+
+                with open(self.full_pointer_file_path, 'w') as f:
+                    f.write(etree.tostring(root, pretty_print=True))
+
+        except Exception, e:
+            LOGGER.info('Attempt to move package %s to location %s failed: %s', self.uuid, destination_location_uuid, str(e))
```

Comment:
`except Exception, e` syntax is deprecated, please use `except Exception as e` if you need it. However, in this case you can use the logging module's `exc_info=True` to automatically add a traceback to the log, which is even better.